google/cloud/storage/internal/retry_client.cc

```diff
@@ -120,14 +122,24 @@ Status ValidateCommittedSize(UploadChunkRequest const& request,
     os << __func__ << ": the server indicates that "
        << response.committed_size.value_or(0) << " bytes are committed "
        << " but given the current request no more than "
-       << expected_committed_size << " should be."
-       << " This could be caused by multiple applications trying to use the"
-       << " same resumable upload, but could be a bug in the library or"
-       << " the service. If you believe this is a bug, please contact support"
-       << " or report the bug at"
-       << " https://github.com/googleapis/google-cloud-cpp/issues/new"
-       << " Include as much information as you can including this message";
-    os << ", session_id=" << request.upload_session_url();
+       << expected_committed_size << " are expected be."
+       << " Most likely your application resumed an upload, and the client"
+       << " library queried the service to find the current persisted bytes."
+       << " In some cases, the service is still writing in data in the"
+       << " background and conservatively reports the bytes persisted."
+       << " Then the next write reports a much higher number of bytes"
+       << " persisted than expected. It is not possible for the client library"
+       << " to recover from this situation. The application needs to resume the"
+       << " upload."
```

Comment:
add a `\n` ? Or is that a bad idea in error messages?

Reply:
I want to avoid those in error messages.